Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection_test.py

```diff
@@ -383,6 +383,20 @@ def test_check_limit_and_offset_values_limit_zero(mocker):
     "isOnlyNetworkUser": "false"
 }
 
+FILE_STATISTICS_API_RESPONSE = {
+    '@odata.context': 'https://api.security.microsoft.com/api/$metadata#microsoft.windowsDefenderATP.api.InOrgFileStats',
+    'sha1': '0991a395da64e1c5fbe8732ed11e6be064081d9f',
+    'orgPrevalence': '14850',
+    'organizationPrevalence': 14850,
+    'orgFirstSeen': '2019-12-07T13:44:16Z',
+    'orgLastSeen': '2020-01-06T13:39:36Z',
+    'globalPrevalence': '705012',
+    'globallyPrevalence': 705012,
```

Comment:
worth a comment here too

Reply:
Comment added in commit 18aefe6f94ba59b7039437d4b010a31d29cf5183 for the "duplicate" fields in the API response